src/ccontrol/UserQueryResources.h

```diff
@@ -83,7 +83,7 @@ class UserQuerySharedResources {
     std::shared_ptr<qmeta::QProgressHistory> queryProgressHistory;
     std::shared_ptr<qmeta::QMetaSelect> qMetaSelect;
     std::shared_ptr<qproc::DatabaseModels> databaseModels;
-    qmeta::CzarId czarId;  ///< Czar ID in QMeta database
+    CzarId czarId;  ///< Czar ID in QMeta database
```

Comment:
The comment "Czar ID in QMeta database" is meaningless (redundant) as the role of the member is completely determined by the name of its type. We have only one concept of czar identity in Qserv.